samples/GuestConfiguration/audit-installed-application-linux/README.md

```diff
@@ -0,0 +1,42 @@
+# Audit for installed application - Linux
+
+This policy audits the results of a check for applications installed in Linux virtual machines.
```

Comment:
This sentence doesn't make sense as worded

Reply:
simplified